app/frontend/src/stories/__mocks__/store.ts

```diff
@@ -0,0 +1,6 @@
+import rootReducer from "../../reducers";
+import { store } from "../../store";
+
+store.replaceReducer(rootReducer);
```

Comment:
What does this do?

Reply:
originally the store came with rootReducer and persister. That persist logic is very annoying when testing. So I replaced the reducer with just the rootReducer.